### ISSUE TITLE
Data run isn't working

### ISSUE THREAD

**Issue by Iuhegi** [MinecraftForge/MinecraftForge#5867]
**Minecraft Version:** 1.14.3

**Forge Version:** 27.0.9

**Full Log:** https://gist.github.com/LukeGi/b9180b69e8af04f2f5776c44b85f31dd

**Steps to Reproduce:**
 1. Download and setup fresh mdk
 2. Add the following to the runs:
```gradle
data {
        workingDirectory project.file('run')

        // Recommended logging data for a userdev environment
        property 'forge.logging.markers', 'SCAN,REGISTRIES,REGISTRYDUMP'

        // Recommended logging level for the console
        property 'forge.logging.console.level', 'debug'

        mods {
            examplemod {
                source sourceSets.main
            }
        }
    }
}
```
3. refresh gradle project
4. run the `runData` gradle task
5. you get a crash

**Description of bug:**
There is a crash because of duplicate keys...

**Comment by primetoxinz** [comment]
Can reproduce with same environment and settings

**Comment by Iuhegi** [comment]
Fixed in c284de366b6e1ae2396f094892fd4be3d1a5b8d0
